src/main/java/pl/edukacyjni/user/UserController.java

```diff
@@ -15,7 +15,19 @@ public class UserController {
     private final UserService userService;
 
     @GetMapping
-    public List<User> getAllUsers() {
+    public List<UserDTO> getAllUsers() {
         return userService.getAllUsers();
     }
+
+    @GetMapping("/{id}")
```

Comment:
That will crash with another PR but that's exactly what I wanted to achieve in this course :D So let's leave it for now and we will discuss it together